build.gradle

```diff
@@ -87,12 +87,6 @@ dependencies {
 	implementation group: "org.webjars", name: "webjars-locator-core", version: "0.46"
 	runtimeOnly group: "org.postgresql", name: "postgresql", version: "42.2.18"
 
-	// Tracing deps
```

Comment:
at the top of this file we also declare the versions for these libraries, can we remove that block as well?

Reply:
Good call, yes.